jbmc/regression/jbmc/simplify-classid-of-interface/test_no_simplify_vccs.desc

```diff
@@ -1,11 +0,0 @@
-CORE
```

Comment:
If we are removing the `.desc` file, should we remove the Java file as well or is it used for other things?

Reply:
There are other tests in this directory that are still using this Java code and that do pass as expected.